include/libopencm3/stm32/fdcan.h

```diff
@@ -0,0 +1,1420 @@
+/** @defgroup fdcan_defines FDCAN defines
+@ingroup STM32H7_defines
+@brief <b>libopencm3 Defined Constants and Types for STM32 FDCAN </b>
+@version 1.0.0
+LGPL License Terms @ref lgpl_license
+*/
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef LIBOPENCM3_STM32_FDCAN_H
+#define LIBOPENCM3_STM32_FDCAN_H
+
+#include <libopencm3/cm3/common.h>
+#include <libopencm3/stm32/memorymap.h>
+
+/**@{
+ * @defgroup fdcan_reg_base CAN register base address
+ * @ingroup fdcan_defines
+ * @{*/
+#define FDCAN1			FDCAN1_BASE
+#define FDCAN2			FDCAN2_BASE
```

Comment:
I think these sorts of defines are normalyl in the part specific header, like h7/fdcan.h?

Reply:
Happy to put them there; but it looks like `can.h` follows this same pattern of defining the device define in the file and deferring the effort to the `libopencm3/stm32/memorymap.h` header to sort things out.

Any thoughts?

Reply:
Resolved this by not using can.h as a template. Started following other modules which means that this did get moved to the H7 dir and out of the common.